api/src/main/java/org/infinispan/api/sync/SyncList.java

```diff
@@ -0,0 +1,27 @@
+package org.infinispan.api.sync;
+
+import java.util.Collection;
+
+/**
+ * @since 15.0
+ **/
+public interface SyncList<V> {
+
+   String name();
```

Comment:
This is redundant if we associate data structure per key, as the key is the name.